rfcs/cmt_input.md

```diff
@@ -0,0 +1,52 @@
+# Cmt files - both as input and output
+
+This RFC asks for two separate additions to the compiler front end. They are
+technically independent, but mostly useful together. So I decided to bundled
+them.
+
+The first request is to type check compilation units and produce `.cmt` files:
+
+```
+$ ocamlc -c foo.ml -o foo.cmt
+```
+
+The second request is to take `.cmt` files as input to produce object files:
+
+```
+$ ocamlc -c foo.cmt -o foo.cmo
+$ ocamlopt -c foo.cmt -o foo.cmx
```

Comment:
The `-o` should also be optional here, `ocamlc -c foo.cmt bar.cmt` producing `foo.cmo` and `bar.cmo` in the same way as `ocamlc -c foo.ml bar.ml` does.

Reply:
Sounds good.